comments-bundle/tests/Cron/PurgeSubscriptionsCronTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CommentsBundle\Tests\EventListener;
```

Comment:
```suggestion
namespace Contao\CommentsBundle\Tests\Cron;
```

The namespace seems to be wrong.